.github/workflows/tests-weekly.yaml

```
@@ -0,0 +1,103 @@
+name: weekly-tests
+on:
+  schedule:
+    - cron: '0 0 * * 1'
```

Because cron schedules are unreadable:

```suggestion
    # Every Monday at 00:00 UTC
    - cron: '0 0 * * 1'
```